### PR TITLE
Fix for virsh_capabilities when tcg is disabled on power

### PR DESCRIPTION
**Pull request by bskjois**
Signed-off-by: Kowshik Jois B S <kowsjois@linux.ibm.com>

Description: virsh URI doesn't discriminate between TCG and KVM, so it contains the string "qemu" in both the cases. Which is correct. But when TCG is disabled, there will not be a domain type = 'qemu'.

The correct way to check this is to parse the output of `$QEMU_BIN --accel help` and cross check that with what `virsh capabilities` shows.

### CODE REVIEW

**Comment by bskjois** [comment]
Logs after applying the patch: 
```
 (2/4) type_specific.io-github-autotest-libvirt.virsh.capabilities.no_option: STARTED
 (2/4) type_specific.io-github-autotest-libvirt.virsh.capabilities.no_option: PASS (6.43 s)
```

File: libvirt/tests/src/virsh_cmd/host/virsh_capabilities.py [hunk 1]
```diff
@@ -80,10 +80,16 @@ def compare_capabilities_xml(source):
...
+                    if (arch == "ppc64" or arch == "ppc64le"):
+                        tcg_check = process.run("qemu-system-ppc64 --accel help",
+                                shell=True).stdout_text.split('\n')
+                        if "tcg" not in tcg_check:
+                            logging.info("expected to fail as tcg is disabled")
+                        else:
+                            test.fail("domain type '%s' is not matched"
+                                    " under arch '%s' in "
+                                    "capabilities_xml" %
+                                    (uri_type, arch))
```
Comment (chunfuwen) [inline libvirt/tests/src/virsh_cmd/host/virsh_capabilities.py:92]: I assume here it miss another 'else' , which arch is not "ppc64" or "ppc64le".

Comment (bskjois) [reply libvirt/tests/src/virsh_cmd/host/virsh_capabilities.py:92]: Thanks for finding that out @chunfuwen 
Have addressed it in a new commit. Please take a look.

**Review by chunfuwen — DISMISSED** [review]
LGTM

**Comment by chunfuwen** [comment]
There are some travis CI failure , please fix them.
Python library autopep8 not installed. Please install it if you want to use --fix
/home/runner/work/tp-libvirt/tp-libvirt/libvirt/tests/src/virsh_cmd/host/virsh_capabilities.py:90:37: E128 continuation line under-indented for visual indent
Style: /home/runner/work/tp-libvirt/tp-libvirt/libvirt/tests/src/virsh_cmd/host/virsh_capabilities.py FAIL (FIX NOT OK)
/home/runner/work/tp-libvirt/tp-libvirt/libvirt/tests/src/virsh_cmd/host/virsh_capabilities.py:91:37: E128 continuation line under-indented for visual indent
/home/runner/work/tp-libvirt/tp-libvirt/libvirt/tests/src/virsh_cmd/host/virsh_capabilities.py:92:37: E128 continuation line under-indented for visual indent
/home/runner/work/tp-libvirt/tp-libvirt/libvirt/tests/src/virsh_cmd/host/virsh_capabilities.py:95:33: E128 continuation line under-indented for visual indent
/home/runner/work/tp-libvirt/tp-libvirt/libvirt/tests/src/virsh_cmd/host/virsh_capabilities.py:96:33: E128 continuation line under-indented for visual indent
/home/runner/work/tp-libvirt/tp-libvirt/libvirt/tests/src/virsh_cmd/host/virsh_capabilities.py:97:33: E128 continuation line under-indented for visual indent

**Comment by bskjois** [comment]
@chunfuwen Fixed Ci failures. Please take a look

**Comment by bskjois** [comment]
@chunfuwen Fixed Ci failures. Please take a look

**Comment by bskjois** [comment]
@chunfuwen Could you please review and let me know if I need to do any more changes?

**Review by PraveenPenguin — APPROVED** [review]
LGTM

**Comment by bskjois** [comment]
@chunfuwen I've fixed the Ci failures and merged commits to a single one. Please take a look

**Comment by bskjois** [comment]
@chunfuwen , @dzhengfy Could you please help me merging this patch as 2 LGTM has been labled.

**Review by harihare — APPROVED** [review]
LGTM

**Comment by bskjois** [comment]
@chunfuwen , @dzhengfy any changes required on this PR to get it merged?

**Comment by bskjois** [comment]
@Yingshun Could you please help me merging this patch as 3 LGTM has been labled.

**Pull request merged**